sdstore/sdstore_test.go

```diff
@@ -357,43 +357,6 @@ func TestUploadZipRetry(t *testing.T) {
 	os.Remove("emitterdata_md5.json")
 }
 
-func TestDownload(t *testing.T) {
```

Comment:
I removed this test because Download no longer returns the entire contents as a `[]bytes` (that behavior wasn't used anywhere except in unit tests). 

Download is still tested elsewhere e.g. `TestDownloadWriteBack`.